elements/pfe-codeblock/src/pfe-codeblock.js

```diff
@@ -0,0 +1,269 @@
+// Import polyfills: Array.includes
+import "./polyfills--pfe-codeblock.js";
+
+import PFElement from "../../pfelement/dist/pfelement.js";
+import Prism from "prismjs";
+
+const observerConfig = {
+  childList: true,
+  subtree: true,
+  characterData: true
+};
+
+class PfeCodeblock extends PFElement {
+  static get tag() {
+    return "pfe-codeblock";
+  }
+
+  static get meta() {
+    return {
+      title: "Codeblock",
+      description: "Render code in a styled and fromatted way"
```

Comment:
Can you update this to "Render code in a styled and formatted way"? Formatted is misspelled.